### PR TITLE
Feature/personal info form

### LINKED ISSUE #12: Step 1 Component: Personal INformation
Build the form in personal information as a component that could be used across all pathways.

### PR DESCRIPTION
**Pull request by kathyychenn**
## Tracking Info

Resolves #12

## Changes

- created important facts section
- created formSection component to create individual sections
- created infoForm component for first step of path

## Testing

- tested long text for overflow, etc.

## Confirmation of Change

Screen Shot 2024-03-11 at 7 56 51 PM

### CODE REVIEW

**Review by emmaz12 — APPROVED** [review]

**Review by Madhav182003 — APPROVED** [review]
Ran the code on my personal device. Everything is working as it should. There is no text overflow. Also ran lint check and everything is fine.

File: frontend/src/components/Step2.tsx [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+              >
+                <img src={hovered ? deleteIconHovered : deleteIcon} alt="deletepng" />
+                <span className={styles.deleteText}>Delete</span>
+              </button>
+            </div>
+            <div className={styles.formSectionContainer}>
+              <div className={styles.inputBox}>
+                <label className={styles.inputTitle}>
+                  Courses<span className={styles.boldRed}>*</span>
+                  <select className={styles.inputText} id="drop" required>
```
Comment (Miyuki-L) [inline frontend/src/components/Step2.tsx:385]: duplicated id. with the select for national exam

**Review by Miyuki-L — CHANGES_REQUESTED** [review]
Good work. Just address the compents and I think we'll be good

File: frontend/src/components/Step2.tsx [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+            <img src={plus} alt="buttonpng" height="14px" />
+          </button>
+        </div>
+        {courses.map((course, index) => (
+          <div className={styles.subSection} key={course.id}>
+            <div className={styles.titleContainer}>
+              <h3 className={styles.subTitle}>{"Course " + course.id}</h3>
+              <button
+                className={styles.deleteButton}
+                onMouseEnter={() => {
```
[the remark this answers is not in the document]
Comment (kathyychenn) [reply frontend/src/components/Step2.tsx:367]: sorry, what should i be fixing here?

File: frontend/src/components/Step1.tsx [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+                className={styles.inputText}
+                type="text"
+                placeholder="Enter 2nd Last Name Here"
+              />
+            </label>
+          </div>
+          <div className={styles.inputBox}>
+            <label className={styles.inputTitle}>
+              Gender<span className={styles.boldRed}>*</span>
+              <select className={styles.inputText} id="drop" defaultValue={"default"} required>
```
Comment (Miyuki-L) [inline frontend/src/components/Step1.tsx:118]: duplicate id

File: frontend/src/components/Step1.tsx [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+                type="text"
+                placeholder="Enter Email Address"
+                required
+              />
+            </label>
+          </div>
+          <div className={styles.inputBox}>
+            <label className={styles.inputTitle}>
+              Phone Type<span className={styles.boldRed}>*</span>
+              <select className={styles.inputText} defaultValue={"default"} id="drop" required>
```
Comment (Miyuki-L) [inline frontend/src/components/Step1.tsx:158]: duplicate ID

**Review by Miyuki-L — COMMENTED** [review]
A few more changes to make and we'll be good

**Review by Miyuki-L — APPROVED** [review]

**Pull request merged**